cdk/lib/prism.ts

```diff
@@ -23,33 +22,32 @@ import {
 	Peer,
 } from 'aws-cdk-lib/aws-ec2';
 
-interface PrismEc2AppProps extends Omit<GuStackProps, 'description' | 'stack'> {
+interface PrismProps extends Omit<GuStackProps, 'description' | 'stack'> {
 	domainName: string;
 	minimumInstances: number;
 }
 
-export class PrismEc2App extends GuStack {
-	private static app: AppIdentity = {
-		app: 'prism',
-	};
-
-	constructor(scope: App, id: string, props: PrismEc2AppProps) {
+export class Prism extends GuStack {
```

Comment:
The `riff-raff.yaml` generator [uses the class name](https://github.com/guardian/cdk/blob/70de4fb237e5f9ff94dc5b54270efb6343448a47/src/riff-raff-yaml-file/deployments/cloudformation.ts#L39) for the `app` property of the `cloud-formation` deployment type. Rename `PrismEc2App` to `Prism` as the app tag is `prism` not `prism-ec2-app`.